samwell/sam/__init__.py

```diff
@@ -177,7 +177,7 @@ class SamFileType(enum.Enum):
 
     def __init__(self, mode: str, ext: str) -> None:
         self.mode = mode
-        self.ext = ext
+        self.ext = ext # type: ignore
```

Comment:
I don't know why this is necessary, but mypy doesn't seem to like calling it `ext`, and rather than changing the name I think it's best to tell mypy to ignore it.

Reply:
import `from typing_extensions import Final` then type these with `Final[str]`